examples/real-world/store/configureStore.prod.js

```diff
@@ -1,14 +1,11 @@
 import { createStore, applyMiddleware } from 'redux'
 import createSagaMiddleware, { END } from 'redux-saga'
+import sagaMonitor from 'saga-monitor-example'
```

Comment:
do we plan to publish this?

Reply:
we can actually, don't see a lot of values tho. most likely ppl want to use something more advanced, like `redux-saga-devtools `